src/remark/code-block-snippets.js

```diff
@@ -24,8 +24,16 @@ let mdFilePath = "";
  * @typedef {import('mdast').Code} CodeNode
  * @param {Readonly<Omit<ImageNode, 'type'>> & CodeNode} node
  */
+
+/**
+ * Find paragraph -> image pattern and perform the replacement.
+ */
 function visitor(node) {
```

Comment:
New jsdoc breaks the param type declaration above.

Move most of the first jsdoc to plugin function
```js
/**
 * This plugin searches for ![code](source-file) pattern and replaces it with the code block
 * containing the content of the source-file. It also supports importing specific lines of code or
 * regions inside the source file defined by #region and #endregion literals.
 *
 * Examples:
 * ![code](../examples/my-example/shell-dump.txt)
 * ![code go](../examples/my-example/src/main.go)
 * ![code go](../examples/my-example/src/server.go "src/server.go")
 * ![code go](../examples/my-example/src/server.go#L3)
 * ![code go](../examples/my-example/src/server.go#L5-L11 "src/server.go")
 * ![code go](../examples/my-example/src/account.go#native-account "Structure of the Native account")
 */
function plugin(options) {
```

and move param type into visitor and update it
```js
/**
 * Find paragraph -> image pattern and perform the replacement.
 *
 * @param {Omit<import('mdast').Paragraph, 'type'> & import('mdast').Code} node
 */
function visitor(node) {
```